common/api.js

```diff
@@ -0,0 +1,184 @@
+const productsData = [
+    {
+        id: 'bag',
+        image: '../assets/bag.jpg',
+        name: 'Bag',
+        desc: 'Luke would be proud.',
+        clickedCount: 0,
+        shownCount: 0
```

Comment:
It looks like you're keeping track of the votes in a different array not this one so I there lines in every object aren't necessary.